### PR TITLE
Set default effect to coverflow widget

### PR DESCRIPTION
**Pull request by wszczepanski97**
[Issue]: #387
[Problem]: Default effect is not set and after selecting images there is still grey background displayed.
[Solution]: Create a function addImagesToCoverflowWidget which sets initial value.

### CODE REVIEW

**Comment by DohyungLim** [comment]
@wszczepanski97 
The CoverFlow widget looks normal on the initial screen.

But I checked the save-preview and came back to the Container and unselected the CoverFlow widget.
And again I chose the CoverFlow widget but I don't see any attributes that change the effect.
Please check the issue.

File: design-editor/src/panel/property/attribute/attribute-element.js [hunk 1]
```diff
@@ -606,10 +606,44 @@ class Attribute extends DressElement {
...
+     * @param {Event} e
+     */
+	addImagesToCoverFlowWidget(event) {
+		let str = '',
+			iframeElement,
+			element;
+		const editor = AppManager.getActiveDesignEditor(),
+			model = editor.getModel(),
+			name = event ? event.target.name : 'effect',
+			value = event ? event.target.value : 'carousel';
```
Comment (DohyungLim) [inline design-editor/src/panel/property/attribute/attribute-element.js]: why default effect is carousel?
default effect is coverflow that correct.

**Comment by wszczepanski97** [comment]
> The selected images are not saved in the images folder within the project. Also, the images all look broken.
@DohyungLim please check is it working now.

> And I checked the save-preview and came back to the Container and unselected the CoverFlow widget.
> And again I chose the CoverFlow widget but I don't see any attributes that change the effect.

There is an issue https://github.com/Samsung/TAU-Design-Editor/issues/359 which is a next step to make coverflow work correct. For now I focus on the first adding of the photos.

File: design-editor/src/panel/property/attribute/attribute-element.js [hunk 2]
```diff
@@ -595,20 +595,59 @@ class Attribute extends DressElement {
...
+			this._$coverflowFileSize.show();
+		});
+	}
+
+	/**
+     * Add images inside <li> tags to build coverflow widget
+     * @param {Event} e
+     */
+	addImagesToCoverFlowWidget(event) {
+		let str = '',
```
Comment (korneliakobiela) [inline design-editor/src/panel/property/attribute/attribute-element.js]: I think these `let` variables aren't really needed in this place. 
- `str` is used only in loop and can be initialized there. 
- both `iframeElement` and `element` are used in if (selectFileLen) scope and can be initialized then. 
I think transformations like this allow make shorter and more readable functions.

**Comment by DohyungLim** [comment]
@wszczepanski97 
next step to #359, please check that. this PR is no problem. I accepted.

**Pull request merged**